final_task/rss_reader/classes/logger.py

```diff
@@ -0,0 +1,19 @@
+import logging
+import sys
+
+
+class logger:
+    '''
+    Uses standart logging lib to implement custom logger that can be used
+    within whole project
+    '''
+    is_log_on = False
+    logging.basicConfig(stream=sys.stdout, format='%(levelname)s:%(message)s', level=logging.DEBUG)
+        
+    @staticmethod
+    def log(msg):
+        '''
+        Receives message and prints it into stdout if logging is turned on
+        '''
+        if logger.is_log_on:
+            logging.debug(msg)
```

Comment:
You do not really need class for this. More so if you have just one static method in it. Simple function will suffice.

Reply:
Done